src/utils.jl

```diff
@@ -52,11 +52,436 @@ function uniquenames(names, newnames)
 end
 
 function projjsonstring(code; multiline=false)
-  spref = spatialref(code)
-  wktptr = Ref{Cstring}()
-  options = ["MULTILINE=$(multiline ? "YES" : "NO")"]
-  GDAL.osrexporttoprojjson(spref, wktptr, options)
-  unsafe_string(wktptr[])
+  wkt2str = CoordRefSystems.wkt2(code)
+  wkt2_to_projjson(wkt2str, multiline=multiline)
+end
+
+# Constants for WKT2 to PROJJSON conversion
+const DEG_TO_RAD = π / 180
+
+function wkt2_to_projjson(wkt2str; multiline=false)
```

Comment:
Please remove underscore from function and variable names to match our code style. Try to pick names that are easy to read without underscore.